packages/components/src/text/stories/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Internal dependencies
+ */
+import { Text } from '../Text.styles';
+
+export default {
+	title: 'components|Text',
```

Comment:
⛏ Tiny nit pick

> components

It doesn't **appear** to matter in Storybook. The other places we label it uses a capital `C`